src/PollerThread.tpp

```diff
@@ -0,0 +1,70 @@
+//=============================================================================
+//
+// file :        PollerThread.tpp
+//
+// description : PollerThread template methods implementation.
+//
+// project :	Tango Device Server
+//
+// $Author: graziano $
+//
+// $Revision: 1.5 $
+//
+// $Log: PollerThread.h,v $
+//
+//
+//
+// copyleft :    European Synchrotron Radiation Facility
+//               BP 220, Grenoble 38043
+//               FRANCE
+//
+//=============================================================================
+
+#ifndef _POLLER_THREAD_TPP
+#define _POLLER_THREAD_TPP
+
+#include "HdbDevice.h"
+#include "Consts.h"
+
+/**
+ * @author	$Author: graziano $
+ * @version	$Revision: 1.5 $
+ */
+
+//	constants definitions here.
+//-----------------------------------------------
+
+namespace HdbEventSubscriber_ns
+{
+
+    template<typename T>
+    void PollerThread::push_events(const std::string attr_name, T* data)
```

Comment:
I don't see these template used. Can the whole file be removed?

Reply:
the file is not even included… you're right, it is gone. Thanks